doc/fileformat.rst

```diff
@@ -0,0 +1,74 @@
+OSI File Format
+----------------
+
+To save multiple serialized OSI messages into one trace file we use the length of each OSI message and save it before the actual OSI message. 
+The length is represented by the first four bytes which are a little endian unsigned int that represents the length of the followed message, not including the integer itself.
+If you happen to have a trace file which uses ``$$__$$`` separation you can convert it to the official OSI trace file by running ``python3 txt2osi.py -d trace.txt``.
+
+To read to content of a serialized txt/osi trace file we also provide a converter ``osi2read.py``.
+See the usage below:
+
+.. code-block:: bash
+
+    python3 osi2read.py -d trace.osi -o readable_trace
+    python3 osi2read.py -d trace.txt -f separated -o readable_trace
+
+which outputs a ``readable_trace.txth`` which can be opened by any text editor.
```

Comment:
txth?

Please add a "warning" for large files as in the PR description

Reply:
txth just stands for "(**t**)e(**x**)(**t**) (**h**)uman readable" for distinguishing between serialized txt and readable txth. Will add the warning good point.